### PR TITLE
Load HTTP Auth variables from environment.

### PR DESCRIPTION
**Pull request by sashax**
Why:
 - 'Cuz  they oughtn't be checked in. Duh.

How:
 - Use dotenv gem to load from .env file.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -3,13 +3,14 @@
...
 require 'yaml'
-
+require 'dotenv'
 
 class RelyingParty < Sinatra::Base
   enable :sessions
 
   use Rack::Auth::Basic, "Restricted" do |username, password|
-    username == '18f' and password == 'Trust But Verify'
+    Dotenv.load
```
Comment (sbc100) [inline app.rb]: Do this just once at startup?

**Comment by sbc100** [comment]
lgtm % comments

File: app.rb (hunk 1 above)
Comment (sashax) [inline app.rb]: good point, Sam. Done.

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,2 @@
+SP_PASS=password_for_SP
+SP_NAME=username_for_SP
```
Comment (sbc100) [inline .env.example]: new line at EOF

File: app.rb [hunk 3]
```diff
@@ -3,13 +3,15 @@
 require 'sinatra/base'
 require 'onelogin/ruby-saml'
 require 'yaml'
+require 'dotenv'
 
+Dotenv.load
```
Comment (sbc100) [inline app.rb]: Seems reasonable I guess.  Is this normal in ruby?

File: app.rb [hunk 4]
```diff
@@ -3,13 +3,15 @@
 require 'sinatra/base'
 require 'onelogin/ruby-saml'
 require 'yaml'
+require 'dotenv'
```
Comment (sbc100) [inline app.rb]: Alpha sort?

**Comment by sbc100** [comment]
Squash the third fixup commit maybe?

otherwise lgtm.   Feel free to submit.  My comments are mostly just nits.

**Pull request merged**